Gemfile

```diff
@@ -5,7 +5,7 @@ gem 'activerecord-import'
 gem 'activerecord-session_store'
 gem 'acts_as_list'
 gem 'addressable'
-gem 'autoprefixer-rails', '7.2.2'
```

Comment:
From Travis it looks like this version of `autoprefixer-rails` no longer supports `therubyracer` as a JS Runetime

Reply:
Yeah, I know, that's the hold up for this.